dashboard/app/controllers/registrations_controller.rb

```diff
@@ -116,7 +116,11 @@ def respond_to_account_update(successfully_updated, flash_message_kind = :update
         format.any {head :no_content}
       else
         format.html {render "edit", formats: [:html]}
-        format.any {head :unprocessable_entity}
+        format.any do
+          render status: :unprocessable_entity,
```

Comment:
The opaque erroring here has bothered me for *sooooo* long, but I always forgot about it when I was working nearby - thank you for setting this up!